org.civicrm.reqangular/src/common/services/session.js

```diff
@@ -42,4 +42,4 @@
       };
     }]);
   });
-})(CRM, Drupal);
+})(CRM, window.Drupal);
```

Comment:
The service returns `contact_id `, I think it should be named as `loggedInUserId`, other wise we are going to get same confusion again.